src/_components/ForNPOs/Hero.jsx

```diff
@@ -0,0 +1,37 @@
+import React from "https://esm.sh/react";
+
+export default function Hero({ comp }) {
+  return (
+    <section className="grow flex flex-col min-h-[700px] lg:flex-row lg:items-center relative overflow-hidden bg-primary text-white">
+      <div className="px-6 md:px-12 lg:px-32 py-8 md:py-16 lg:py-0">
+        <h1 className="font-bold text-3xl md:text-4xl lg:text-5xl leading-snug md:leading-[1.3] tracking-wide mb-4 md:mb-6 lg:mb-16 text-center lg:text-left">
+          Non-Profit Organizations
+        </h1>
+        <p className="text-lg md:text-xl lg:text-2xl leading-snug md:leading-[1.3] mb-6 md:mb-10 lg:mb-14 text-center lg:text-left">
+          We are passionate about leveraging our technical and
+          <br />
+          consulting skills to support your organization’s mission,
+          <br />
+          free of charge!
+        </p>
+        <div className="flex flex-col md:flex-row md:justify-center lg:justify-start space-y-4 md:space-y-0 md:space-x-6">
+          <comp.Button
+            style="h-10 px-6 md:px-8 min-w-[191px] rounded-md border border-white justify-center items-center gap-2 text-white font-semibold text-base md:text-lg inline-flex"
```

Comment:
I believe hard setting `h-10` causes a slight issue with resizing so if the page shrinks to a certain breakpoint it causes the text to overflow out of the button (see image below). You could fix that by not setting the height but adding `py-2` so the button just has a consistent vertical padding too. Additionally, if you want the next itself to be centered in this weird state, you can apply `text-center` div too (the flex centering only centers the text container not aligns the text itself)

<img width="300" alt="Screenshot 2024-11-06 at 8 45 01 AM" src="https://github.com/user-attachments/assets/6742d4b2-6a85-4c0a-8f0c-7df84856c0a8">